web/api/controller/market_controller.py

```
@@ -1,24 +1,11 @@
 from fastapi import APIRouter
-from web.api.request.inference_request import MarketNewsRequest, MarketSectorRequest
-
+zz
```

Ruff (0.14.5)

Found useless expression. Either assign it to a variable or remove it.

(B018)